### ISSUE TITLE
Partners as admins and users? How does that work?

### ISSUE THREAD

**Issue by MainStreamDanny** [gpsimpact/mainstream_relational_gotv#8]
I've created my own Org, Danny's Test Org in the database. It worked great.

Then I registered as a user for that org. I used the same email address as I put in the contact email for the Org. I was testing the admin_notes field, but it occurs to me that I don't know what the process is for partner admins to get a dashboard page? Clearly, adding them as partners doesn't do it, because they (or me, as the admin entering them) don't have to enter a password.

So I assumed that registering themselves with that org using the same email address would indicate to the system that they are the admin, and should have admin links displayed.

I also assumed that the admin links would be contained in the "admin_notes" field in the org db, but I don't see the sample notes I added when I log in as myself in my own org.

How does this work? Thanks.

**Comment by jakelowen** [comment]
Take a look at the permissions table of the database....

Registering with a new organization will automatically create a link between the user email and the organization (defined in organizations table).  There are currently 3 possible permissions possible in the permissions table:

'READ_ONLY', // Read only permissions
'ADMIN', // can grant/revoke permissions to others
 'AMBASSADOR',

These permissions are not intended to be "one or the other", but "all that apply"..  So a typical ambassador would only have one row with the 'AMBASSADOR' permission, whereas an admin would have both 'AMBASSADOR' and 'ADMIN' rows.

So in short, find the row in the permissions table where your email is given AMBASSADOR status with the Dannys test org.  Now create a row just like that with the ADMIN permission for a total of 2 rows for that given email / org.  

NOTE: You will have to log out and log back in to the website to see you new permissions reflected.

In the same way you will be able to remove a user's permission (ambassador or admin) by simply deleting the relevant row in the DB.

**Comment by MainStreamDanny** [comment]
So "ADMIN" is the level for an org partner contact person? At permission level does the "admin_notes" field display in the interface?
